base/tks/src/main/java/org/dogtagpki/server/tks/servlet/SecureChannelProtocol.java

```diff
@@ -1303,6 +1320,30 @@ public SymmetricKey generateSymKey(String selectedToken) throws EBaseException {
 
     }
 
+    // Will be needed for aes based server side keygen to come
```

Comment:
since this function is not being used for now.  Could you consider commenting it out so that we don't forget to properly review it later when it is used?

Reply:
Will be in next version of pr.